barman/server.py

```diff
@@ -784,10 +805,15 @@ def check_postgres(self, check_strategy):
             check_strategy.result(
                 self.config.name, remote_status.get("streaming"), hint=hint
             )
+
+    def _check_wal_level(self, check_strategy, remote_status, suffix=None):
         # Check wal_level parameter: must be different from 'minimal'
         # the parameter has been introduced in postgres >= 9.0
         if "wal_level" in remote_status:
-            check_strategy.init_check("wal_level")
+            check_name = "wal_level"
+            if suffix is not None:
+                check_name = f"{check_name} ({suffix})"
```

Comment:
```suggestion
            check_name = "wal_level" + ("" if suffix is None else f" ({suffix})")
```

Same comment in regards to a single attribution.